### PR TITLE
SOLVED ASCII CODE OF EACH CHARACTER OF ENTERED STRING PROBLEM

### LINKED ISSUE #3947: ASCII CODE OF EACH CHARACTER OF ENTERED STRING IN Java (GSSOC'21)
### Have you read the [Contributing Guidelines on Pull Requests](https://github.com/TesseractCoding/NeoAlgo/blob/master/CONTRIBUTING.md#reporting-new-issues)?

Yes !

AIM :: **to find out the ASCII CODE OF EACH CHARACTER OF ENTERED STRING IN Java**

## Motivation

As i didn't found ASCII CODE EACH CHARACTER OF ENTERED STRING IN JAVA program, So i am willing to contribute it...

## Pitch

Hi I am GSSoC'21 participant,
"I am looking to work on ASCII CODE OF ENTERED STRING IN JAVA"
I will write the clean and optimized code , by keeping best case time and space complexity in mind and 
with  brief commenting and algorithm.
I will use Java language.

### PR DESCRIPTION
**Pull request by ananddasani**
### Have you read the [Contributing Guidelines on Pull Requests](https://github.com/TesseractCoding/NeoAlgo/blob/master/CONTRIBUTING.md#pull-requests)?

Yes !!

### Description

i have implemented the solution of SOLVED ASCII CODE OF EACH CHARACTER OF ENTERED STRING PROBLEM
so i would request @HarshCasper , @ankitaggarwal23 , @bhav09 to please review my PR.

### Checklist

- [X] I've read the contribution guidelines.
- [X] I've checked the issue list before deciding what to submit.
- [X] I've edited the `README.md` and link to my code.

## Related Issues or Pull Requests
No :)

Fixes #3947

### CODE REVIEW

File: Java/README.md [hunk 1]
```diff
@@ -179,6 +179,7 @@ _add list here_
 - [Factorial Using Big Integer](math/Big_Integer_Factorial.java)
 - [Polynomial Sum & Product using Linked Lists](math/polynomial.java)
 - [Neon number](Neon_number.java)
+- [ASCII_code_of_each_character_of_string](math/ASCII_code_of_each_character_of_string.java)
```
Comment (ankitaggarwal23) [inline Java/README.md]: - Remove Underscores from `[ASCII_code_of_each_character_of_string]`
- `ASCII_code_of_each_character_of_string.java` make it concise and short

File: Java/math/ASCII_code_of_each_character_of_string.java [hunk 2]
```diff
@@ -0,0 +1,39 @@
+//AIM :: to print the ASCII CODE of each character of entered string
```
Comment (ankitaggarwal23) [inline Java/math/ASCII_code_of_each_character_of_string.java]: Add a bit more description

File: Java/math/ASCII_code_of_each_character_of_string.java [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+
+import java.util.Scanner;
+
+public class ASCII_code_of_each_character_of_string {
```
Comment (ankitaggarwal23) [inline Java/math/ASCII_code_of_each_character_of_string.java]: Make it short and concise ASCII_code_of_each_character_of_string

File: Java/math/ASCII_code_of_each_character_of_string.java [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+
+TEST CASE 2
+
+Enter the string :: anand
+a = 97
+n = 110
+a = 97
+n = 110
+d = 100
+
```
Comment (ankitaggarwal23) [inline Java/math/ASCII_code_of_each_character_of_string.java]: Remove extra test case one is enough

File: Java/math/ASCII_code_of_each_character_of_string.java [hunk 5]
```diff
@@ -0,0 +1,39 @@
...
+ */
```
Comment (ankitaggarwal23) [inline Java/math/ASCII_code_of_each_character_of_string.java]: Add a new line at the EOF

**Review by ankitaggarwal23 — CHANGES_REQUESTED** [review]

File: Java/math/ASCII_code_of_each_character_of_string.java (hunk 5 above)
Comment (ananddasani) [reply Java/math/ASCII_code_of_each_character_of_string.java]: okay

**Comment by ananddasani** [comment]
@ankitaggarwal23 i have made all the changes as directed , please review again :)

File: Java/math/AsciiOfString.java [hunk 6]
```diff
@@ -0,0 +1,33 @@
...
+        String str = sc.nextLine();
+
+        // printing all the characters with their ASCII code
+        for (int i = 0; i < str.length(); i++)
+            System.out.println(str.charAt(i) + " = " + (int) str.charAt(i));
+    }
+}
+
+/*
+TEST CASE
```
Comment (atarax665) [inline Java/math/AsciiOfString.java:23]: Add space and time complexity.

**Review by atarax665 — CHANGES_REQUESTED** [review]
Segregate Driver code and the logic part. The main function should only have the driver code.

**Comment by ananddasani** [comment]
@ankitaggarwal23 all set ! 

please review this PR also :)

**Review by ankitaggarwal23 — APPROVED** [review]

**Pull request merged**